code/modules/mob/living/bot/secbot.dm

```diff
@@ -361,6 +361,7 @@
 /mob/living/bot/secbot/proc/calc_path(var/turf/avoid = null)
 	path = AStar(loc, patrol_target, /turf/proc/CardinalTurfsWithAccess, /turf/proc/Distance, 0, 120, id=botcard, exclude=avoid)
 	if(!path.len)
+		path = list()
```

Comment:
This doesn't make sense. If accessing `path.len` doesn't runtime, and the list has no length, why are we swapping it for a new list _now_? The effects of `path = list()` are already a requirement for us to get here.